src/modules/0.0.22/aws_acm_list/index.ts

```diff
@@ -1,4 +1,5 @@
 import { ACM, CertificateDetail, paginateListCertificates } from '@aws-sdk/client-acm';
+import { parse as parseArn } from '@aws-sdk/util-arn-parser';
```

Comment:
interesting addition, I think in other modules we are manually splitting the ARN and this could be a good improvement.